build/README.md

```diff
@@ -0,0 +1,28 @@
+# Finding missing translations
```

Comment:
Might be wise to also add instructions on how to add more languages ^^

Reply:
Oh wasn't aware that we already covered all vanilla languages. In that case we don't need docs for adding new languages cause we don't really have a way of adding them anyway :P